### PR TITLE
e2e-tests - folder structure

### PR DESCRIPTION
**Pull request by rbrtrfl**
For the new Prozesskostenhilfe form, parts of the e2e-tests are to be moved to a shared folder.

### CODE REVIEW

**Review by frederike-ramin — APPROVED** [review]
I love it! The only thing I'd suggest is putting `shared` under `flowPages`:

- flowPages
  - bh
  - pkh
  - shared
- util
- etc

And maybe skip the test in `tests/e2e/flowPages/prozesskostenhilfe/formular/prozesskostenhilfe.formular.spec.ts` instead of commenting it out if it doesn't work yet? (Or don't commit it in the first place :) )

File: tests/e2e/flowPages/beratungshilfe/beratungshilfe.spec.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { test } from "@playwright/test";
-import { testPageToBeAccessible } from "./util/testPageToBeAccessible";
+import { testPageToBeAccessible } from "../../util/testPageToBeAccessible";
 
 test.beforeEach(async ({ page }) => {
   await page.goto("/beratungshilfe");
```
Comment (aaschlote) [inline tests/e2e/flowPages/beratungshilfe/beratungshilfe.spec.ts:5]: This file shouldn't on the pages folder? Technically is not a flow page, but a normal page, right?

File: tests/e2e/flowPages/beratungshilfe/vorabcheck/csrf.spec.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import { test, expect } from "@playwright/test";
 import { csrfCountMax } from "~/services/security/csrf.server";
-import { BeratungshilfeVorabcheck } from "./pom/BeratungshilfeVorabcheck";
-import { CookieSettings } from "./pom/CookieSettings";
+import { BeratungshilfeVorabcheck } from "../../../pom/BeratungshilfeVorabcheck";
+import { CookieSettings } from "../../../pom/CookieSettings";
```
Comment (aaschlote) [inline tests/e2e/flowPages/beratungshilfe/vorabcheck/csrf.spec.ts:4]: I believe this is not in the correct folder, here it's testing the execution of csrf on the Beratungshilfe Vorabcheck pages, so the tests are checking the csrf and not the Beratungshilfe Vorabcheck. Can we move to a new folder, like common?

File: tests/e2e/flowPages/beratungshilfe/vorabcheck/userfeedback.vorabcheck.spec.ts [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import { test, expect, type Page } from "@playwright/test";
-import { BeratungshilfeVorabcheck } from "tests/e2e/pom/BeratungshilfeVorabcheck";
-import { CookieSettings } from "./pom/CookieSettings";
+import { BeratungshilfeVorabcheck } from "../../../pom/BeratungshilfeVorabcheck";
+import { CookieSettings } from "../../../pom/CookieSettings";
```
Comment (aaschlote) [inline tests/e2e/flowPages/beratungshilfe/vorabcheck/userfeedback.vorabcheck.spec.ts:3]: Same here

File: tests/e2e/flowPages/beratungshilfe/vorabcheck/hiddenfields.spec.ts [hunk 4]
```diff
@@ -1,5 +1,5 @@
 import { test, expect } from "@playwright/test";
-import { BeratungshilfeVorabcheck } from "./pom/BeratungshilfeVorabcheck";
+import { BeratungshilfeVorabcheck } from "../../../pom/BeratungshilfeVorabcheck";
```
Comment (aaschlote) [inline tests/e2e/flowPages/beratungshilfe/vorabcheck/hiddenfields.spec.ts:2]: same here

**Review by aaschlote — CHANGES_REQUESTED** [review]
Nice changes, two things to check:

1. You have some lint warnings, can you get rid of them?
2. You are using too much ``../.../...`` to import some files, could you use the shortcut ``test/`` or create a new one for the e2e path in the tsconfig?

Nice work 😃

**Pull request merged**